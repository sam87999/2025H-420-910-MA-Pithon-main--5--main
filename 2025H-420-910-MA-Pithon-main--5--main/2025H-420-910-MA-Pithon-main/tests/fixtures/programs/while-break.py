# Test de Dominic Lemay
x = 5
while x > 0:
    if x == 3:
        break
    print(x)
    x = x - 1
print("done")
