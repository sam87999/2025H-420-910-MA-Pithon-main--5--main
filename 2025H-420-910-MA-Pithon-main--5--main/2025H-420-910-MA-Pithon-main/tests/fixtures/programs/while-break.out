5
4
done
